src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -194,15 +195,16 @@ pub struct Args {
     /// Wildcards in other positions (e.g., "https://*.foo.com" or "https://foo.*.com") have no effect.
     #[structopt(long, env = "CORS_ALLOWED_ORIGIN")]
     cors_allowed_origin: Vec<HeaderValue>,
-    /// Public IP addresses which the cloud environment has configured for
+    /// Public IP or CIDR which the cloud environment has configured for
```

Comment:
It's always a CIDR now, right? It's just that you can specify a /32?

```suggestion
    /// Public CIDR which the cloud environment has configured for
```

Reply:
This actually still allows standard IPs and will just convert them to a CIDR of `/32` this worked out of the box with the `cidr` crate so I went with it. If that's not the case with `ipnet` there's no reason we shouldn't only support cidr ranges.

Reply:
Ah, gotcha!